qemu/tests/numa_cpu.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from virttest import error_context
+from virttest.utils_misc import normalize_data_size
+from virttest.utils_misc import NumaInfo
+
+
+def run(test, params, env):
+    """
+    Assign cpu to numa node with "-numa cpu", check the numa info in monitor
+    and guest os match with the qemu cli
+    """
+
+    def numa_cpu_guest():
+        """
+        The numa cpus in guest os, linux only
+        """
+        numa_info_guest = NumaInfo(session=session)
+        nodes_guest = numa_info_guest.online_nodes
+        numa_cpu_guest = []
+        for node in nodes_guest:
+            numa_cpus = numa_info_guest.online_nodes_cpus[node]
```

Comment:
`NumaInfo` does not have this attribute.
```python
dir(NumaInfo())
ERROR:avocado.utils.software_manager:dnf module for Python is required. Using the basic support from rpm and dnf commands
WARNING:root:Can not find the cpu list information from both numactl and sysfs. Please check your system.
 Error: /sys/devices/system/node/node1/cpulist is not available
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', 'all_nodes', 'distances', 'get_all_node_meminfo', 'get_all_nodes', 'get_node_distance', 'get_online_nodes', 'get_online_nodes_withcpu', 'get_online_nodes_withmem', 'nodes', 'numa_sys_path', 'online_nodes', 'online_nodes_withcpu', 'online_nodes_withcpumem', 'online_nodes_withmem', 'read_from_node_meminfo', 'session']
```

Reply:
A dependent patch is missed: https://github.com/avocado-framework/avocado-vt/pull/2699
Add it in the description now.